python/django/python-hello-world/README.md

```diff
@@ -0,0 +1,72 @@
+# Hello World with Cloud Code
+
+![Architecture Diagram](./img/diagram.png)
+
+"Hello World" is a simple Kubernetes application that contains
+[Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) for a web server and a database, and corresponding
+[Services](https://kubernetes.io/docs/concepts/services-networking/service/). The Deployment contains a
+[Django-based](https://www.djangoproject.com/) web server that simply prints "Hello World".
+
+----
+
+## Table of Contents
+
+### Cloud Code for Visual Studio Code
+
+1. [Getting Started](#getting-started])
+2. What's in the box
+    * .vscode
+      * extensions.json: prompt requesting download of related extensions when opening this project
+      * launch.json: information needed for the debugger to attach to our service
+      * tasks.json: configuration information for Visual Studio Code Tasks
+    * kubernetes-manifests
+      * hello.deployment.yaml: Kubernetes manifest for the Hello World server Deployment
+      * hello.service.yaml: Kubernetes manifest for the Hello World LoadBalancer Service
+      * hello.migration.yaml: Kubernetes manifest for the Postgres database migration job
+      * postgres.development.yaml: Kubernetes manifest for the Postgres database Deployment
+      * postgres.service.yaml: Kubernetes manifest for the Postgres database Cluster IP Service
```

Comment:
why do we need these if it's just "hello-world"?

Reply:
Yeah I agree that it sounds strange for a hello world application to require a database. But looks like Django was designed to produce database-backed web sites.

Django has a command to create a project template "[startproject](https://docs.djangoproject.com/en/2.2/ref/django-admin/#startproject)". In the project created with this command, by default it uses a sqlite database, though the project has nothing. The reason is that Django projects have a built in [admin interface](https://docs.djangoproject.com/en/2.2/ref/contrib/admin/#module-django.contrib.admin) by default, which uses database.

I can remove the admin interface from the template so that databases are not required, but this may look strange for Django developers. Why the admin interface is enabled in the default project template used by startproject, but not in our template?

Reply:
I think that if we can remove the admin interface for hello world, we should do it.
It should be as simple as possible template. For guestbook we should leave admin interface as this will show second level of complexity. Also this will be great opportunity for external developers to see us show both options

Reply:
Sure I will the admin interface.